File: app/src/main/java/com/gustu/logbook/main/interfaces/MainView.java
package com.gustu.logbook.main.interfaces;

import com.gustu.logbook.main.model.kegiatan.Kegiatan;
import com.gustu.logbook.main.model.levelKesulitan.Kesulitan;
import com.gustu.logbook.main.model.levelPrioritas.Priotitas;

import java.util.List;

public interface MainView {
    void _onKesulitanLoad(List<Kesulitan> kesulitanList);
    void _onPrioritasLoad(List<Priotitas> priotitasList);
    void _onKegiatanLoad(List<Kegiatan> kegiatanList);
    void _onFailed(String t);
}

File: app/src/main/java/com/gustu/logbook/main/interfaces/MainView.kt
package com.gustu.logbook.main.interfaces

import com.gustu.logbook.main.model.kegiatan.Kegiatan
import com.gustu.logbook.main.model.levelKesulitan.Kesulitan
import com.gustu.logbook.main.model.levelPrioritas.Priotitas

interface MainView {
    fun _onKesulitanLoad(kesulitanList: List<Kesulitan>)
    fun _onPrioritasLoad(priotitasList: List<Priotitas>)
    fun _onKegiatanLoad(kegiatanList: List<Kegiatan>)
    fun _onDataAdd()
    fun _onDataFailedAdd()
    fun _onFailed(t: String)
}
